.eslintrc.js

```diff
@@ -22,14 +22,53 @@ module.exports = {
         'plugin:vue/vue3-recommended',
         'plugin:vuejs-accessibility/recommended',
     ],
+    rules: {
+        'vue/block-lang': ['error', {
```

Comment:
**question**: why did you place the rules here instead of the vue block in the overrides?

Reply:
@goester-team23 These rules are optional and not previously declared. That is why we declare them for the first time.